manager/src/main/java/com/redhat/service/bridge/manager/dao/ProcessorDAO.java

```diff
@@ -63,10 +62,9 @@ public Processor findByIdBridgeIdAndCustomerId(String id, String bridgeId, Strin
         return singleResultFromList(find("#PROCESSOR.findByIdBridgeIdAndCustomerId", p));
     }
 
-    public List<Processor> findByStatusesAndShardIdWithReadyDependencies(List<BridgeStatus> statuses, String shardId) {
+    public List<Processor> findByStatusesAndShardIdWithReadyDependencies(String shardId) {
```

Comment:
I think this is not anymore `findByStatusesAndShardIdWithReadyDependencies` but something like `findByShardIdWithReadyDependencies`. If you agree feel free to do it in another PR

Reply:
Agreed. Changed in this PR.